lib/Doctrine/ORM/Query/QueryExpressionVisitor.php

```diff
@@ -51,7 +50,7 @@ class QueryExpressionVisitor extends ExpressionVisitor
     /** @var Expr */
     private $expr;
 
-    /** @var mixed[] */
+    /** @var array<int, mixed> */
```

Comment:
Maybe we should use `list` here and in the constructor?

Reply:
replaced here, the property was not from constructor though